tools/model_tools/src/openvino/model_zoo/_common.py

```diff
@@ -135,25 +135,30 @@ def get_package_path(python_executable, package_name):
     return file_path.parent, completed_process.stderr
 
 def get_version():
-    if VERSION_FILE and VERSION_FILE.is_file():
-        with VERSION_FILE.open('r') as version_file:
-            version = version_file.readline().rstrip('\n')
-        return f'commit {version}'
-    else:
-        return 'unknown'
+    try:
+        from openvino.runtime import get_version
+        version = get_version()
```

Comment:
you can get infinity cycle with overloading and mixing inner and outer scope, let's clarify
```suggestion
        from openvino.runtime import get_version as ov_get_version
        version = ov_get_version()
```

Reply:
Applied